nemo_curator/datasets/doc_dataset.py

```diff
@@ -229,8 +229,9 @@ def _read_json_or_parquet(
     input_files: Union[str, List[str]],
```

Comment:
@rjzamora , Curious if you have thoughts around `_read_json_or_parquet` . 

Now that we control the IO code path directly in `dask_cudf`, i am curious if we can just offload the details of this function there directly than doing it here.

Reply:
Discussed offline